#! /usr/bin/env python

PACKAGE='hearbo_cart_msgs'
import roslib; roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()
gen.add("stop", bool_t, 0, "Emergency stop", False)
gen.add("velocity_x",   double_t, 0, "backward forward velocity",0, -50, 50)
gen.add("velocity_y",   double_t, 0, "left right velocity",0, -50, 50)
gen.add("velocity_rot", double_t, 0, "spin velocity",0, -50, 50)
exit(gen.generate(PACKAGE, "CartVelocityPublisherByDynReconf", "CartVelocity"))
